plugins/PrerenderPlugin/README.md

```diff
@@ -50,6 +50,7 @@ Allowed values are as follows:
 - `deep`: A string or array of string conditions, that when met at runtime, should not display the prerendered HTML.
 - `screenTypes`: Array of 1 or more screentype definitions to be used with prerender HTML initialization. See [here](https://github.com/enactjs/enact/blob/master/packages/moonstone/MoonstoneDecorator/screenTypes.json) for an example of the moonstone screenTypes.
 - `fontGenerator`: Module path to a font stylesheet generator for prerendering fontface definitions. See [here](https://github.com/enactjs/enact/blob/master/packages/moonstone/MoonstoneDecorator/fontGenerator.js) for an example of the moonstone font generator.
+- `inlineStartup`:  A flag whether to inline the startup javascript within the HTML or to externalize into separate js files. Defaults to `true`.
```

Comment:
Not a fan of unnecessary negation prefixes on already-length variable names. I don't personally see much of an issue with a default-on config but if necessary I'd probably prefer alternate rewording suggestions, like `externalStartup` or `startupAssets`. Thoughts?

Reply:
The latter suggestions are probably much better.  I do prefer not to have negative names but I do prefer positive flags rather than having to set them to false.  `externalStartup` seems reasonable to me.